wce-ote/plugin/plugin.js

```diff
@@ -1867,6 +1867,10 @@
 							}
 							break;
 						case 'corr':
+							// this if statement added by Cat in Nov 22 because jquery changed the behaviour of the serialization
+							if (!ar.hasOwnProperty('deletion')) {
+								ar.deletion = 'null';
```

Comment:
I guessing this is to prevent line 1895, below, from failing?  Did the behavior of jQuery previous keep an object's properties which were set to null, but now it leaves them out when serializing the object?

Reply:
yes that is right. This is the only instance of this type of select in the OTE so I went with the option that involved changing the least amount of code. In the old jquery it used to serialise into the html as deletion=null which was then made into a string in the code that turns the html inside the OTE into an array.

Reply:
https://jquery.com/upgrade-guide/3.0/#breaking-change-select-multiple-with-nothing-selected-returns-an-empty-array

This is the reference